### PR TITLE
Fixes mostly-inaccessible bottleneck on North Star's SM waste line

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

The injector in the North Star's SM waste line/waste processing chamber is set to 50L/s. This change sets it to the maximum flow rate, 200L/s, the same throughput as a single filter (and filters chained on a single layer).

## Why It's Good For The Game

A single filter has a throughput of 200L/s. A 5-layer setup for the SM, then, has 1000L/s throughput. While someone setting up a 1000L/s cooling system can reasonably be expected to also set up their own exhaust, those using the North Star's built in waste-gas reprocessing system are then bottlenecked to 25% (50L/s) of the capacity of one filter.

This isn't a problem during normal operation of a nitrogen SM, but can make it something of a "newbie trap" for delams, especially if plasma & oxygen are actively combusting within the waste gas lines. Setting the injector to its maximum capacity of 200L/s means a single-layer filter setup can run at full capacity during delams that don't have a lot of gas burning in the lines (and still mean clogs from in-pipe gas burns will clear faster).

The injector is pretty inaccessible - it's behind R-walls and R-plasma windows with no airlocks nor other provisions for accessing the chamber, making it pretty impractical for engineers to get at the injector to manually change its flow rate. In theory a silicon could do it, but, still, it's pretty obscure and I don't think most would ever think to, even during a delam event.

Lastly, this still leaves room for engineers that wish to improve the system by adding their own exhaust - even sticking to a single-layer filter setup, a space vent would still help in the aforementioned cases of combustion in the pipes, and for those creating 1000L/s setups, it's naturally the only option that can keep up (or come even close).

TL;DR: Makes the North Star's SM cooling system less of a newbie trap while still allowing a lot of gameplay around constructing in-round improvements.

## Changelog

:cl:
fix: Fixed bottleneck on North Star's SM waste line (it can now handle the full output of 1 filter per the stock setup).
/:cl:

### CODE REVIEW

**Comment by YehnBeep** [comment]
Ngh. How do I get github to not include commits from the last PR?

**Comment by kinneb** [comment]
> Ngh. How do I get github to not include commits from the last PR?

this is why we keep clean, seperate branches per PR

**Comment by YehnBeep** [comment]
Ok that was slightly scary but I got it I think

**Comment by YehnBeep** [comment]
...Why is birdshot failing I haven't touched it...

**Comment by Cheshify** [comment]
Very odd, could have sworn this was at 200. Good fix.

**Comment by Maurukas** [comment]
> ...Why is birdshot failing I haven't touched it...

CI weirdness. Probably someone else's fault.

**Review by Maurukas — APPROVED** [review]
